static/src/javascripts/projects/commercial/modules/third-party-tags/audience-science-pql.js

```diff
@@ -28,36 +28,52 @@ define([
     }
 
     function onLoad() {
-        var asiPlacements = window.asiPlacements;
-        var segments = storage.local.get(storageKey) || {};
-        // override the global value with our previously stored one
-        window.asiPlacements = segments[section];
-        segments[section] = asiPlacements;
-        storage.local.set(storageKey, segments);
+        isLoaded = true;
+        mediator.emit('commercial:audience-science:loaded');
     }
 
     function getSegments() {
-        var segments = {};
-        var storedSegments = storage.local.get(storageKey);
-        if (
-            config.switches.audienceScienceGateway &&
-            storedSegments &&
-            storedSegments[section]
-        ) {
-            segments = Object.keys(storedSegments[section])
-                .filter(function (placement) {
-                    //keyword `default` written in dot notation throws an exception IE8
-                    return storedSegments[section][placement]['default']; //eslint-disable-line
-                }).map(function (placement) {
-                    return ['pq_' + placement, 'T'];
-                }).reduce(function (result, input) {
-                    result[input[0]] = input[1];
-                    return result;
-                }, {});
-            // set up the global asiPlacements var in case dfp returns before asg
-            window.asiPlacements = storedSegments[section];
+        var placements = window.asiPlacements || {};
+        return Object.keys(placements)
+        .filter(function (placement) {
```

Comment:
lol, I use this style because it's clear (for me) and because if you indent the `.filter` most editors mess up with indentation, so when you press enter they stay on the `.` level instead of going back to `return`. So yeah